internal/pkg/cli/init.go

```diff
@@ -153,22 +142,81 @@ func newInitOpts(vars initVars) (*initOpts, error) {
 		cmd:          command.New(),
 		sessProvider: sessProvider,
 	}
+	deployJobCmd := &deployJobOpts{
+		deployJobVars: deployJobVars{
+			envName:  defaultEnvironmentName,
+			imageTag: vars.imageTag,
+			appName:  vars.appName,
+		},
+		store:        ssm,
+		prompt:       prompt,
+		ws:           ws,
+		unmarshal:    manifest.UnmarshalWorkload,
+		sel:          sel,
+		spinner:      spin,
+		cmd:          command.New(),
+		sessProvider: sessProvider,
+	}
 
 	return &initOpts{
+		initVars:     vars,
 		ShouldDeploy: vars.shouldDeploy,
 
 		initAppCmd:   initAppCmd,
-		initSvcCmd:   initSvcCmd,
 		initEnvCmd:   initEnvCmd,
 		deploySvcCmd: deploySvcCmd,
+		deployJobCmd: deployJobCmd,
 
-		appName:        &initAppCmd.name,
-		svcType:        &initSvcCmd.wkldType,
-		svcName:        &initSvcCmd.name,
-		svcPort:        &initSvcCmd.port,
-		dockerfilePath: &initSvcCmd.dockerfilePath,
+		appName: &initAppCmd.name,
 
 		prompt: prompt,
+
+		setupWorkoadInit: func(o *initOpts, wkldType string) error {
+			wlInitializer := &initialize.WorkloadInitializer{Store: ssm, Ws: ws, Prog: spin, Deployer: deployer}
+			wkldVars := initWkldVars{
+				appName:        *o.appName,
+				wkldType:       wkldType,
+				name:           vars.svcName,
+				dockerfilePath: vars.dockerfilePath,
+				image:          vars.image,
+				port:           vars.port,
+				schedule:       vars.schedule,
+				retries:        vars.retries,
+				timeout:        vars.timeout,
+			}
+			switch wkldType {
+			case manifest.ScheduledJobType:
+				opts := initJobOpts{
+					initWkldVars: wkldVars,
+
+					fs:     &afero.Afero{Fs: afero.NewOsFs()},
+					init:   wlInitializer,
+					sel:    sel,
+					prompt: prompt,
+				}
+				o.initWlCmd = &opts
+				o.initWkldVars = &opts.initWkldVars
+			case manifest.LoadBalancedWebServiceType:
+				fallthrough
```

Comment:
In the Go time podcast, they were talking how fallthrough is an anti-pattern as it's so rarely used and leads to confusion.

Instead, can we do this:
```go
switch t := wkldType; {
case t == manifest.LBWebServiceType || t == manifest.BackendService:

case t == maniefst.ScheduledJobType:
}
```